lib/tasks/deploy.rake

```diff
@@ -37,8 +43,22 @@ namespace :deploy do
     check_gem! 'rails_migrate_mutex' unless ENV['NO_DB']
     check_gem! 'rack-timeout'
 
-    sh "cp -r #{template_dir}/.??* ."
-    sh "cp -r #{template_dir}/* ."
+    Dir.foreach(template_dir) do |item|
+      next if item == '.' or item == '..'
```

Comment:
Haha, ruby 2.5 has a method([`each_child`](https://ruby-doc.org/core-2.5.0/Dir.html#method-c-foreach)) that skips these entries :)

Reply:
yeah sadly we cannot yet rely on the gem users to use 2.5